server/webdriver/shared_tests/ranking_test.py

```diff
@@ -53,12 +53,11 @@ def test_northamerica_population(self):
     self.assertEqual(x_text[0].text, 'United States of America')
     self.assertEqual(x_text[-1].text, 'Montserrat')
 
-  def test_energy_consumption_bottom_ranking_hi(self):
+  def test_population_bottom_ranking_hi(self):
     """Test translations are displayed correctly in hindi, as well as bottom rankings rendered correctly."""
     self.driver.get(
         self.url_ +
-        '/ranking/Amount_Consumption_Energy_PerCapita/Country/?h=country%2FIND&unit=kg&hl=hi&bottom='
-    )
+        '/ranking/Count_Person/Country/?h=country%2FIND&unit=kg&hl=hi&bottom=')
```

Comment:
nit: maybe remove the unit=kg because that doesn't super make sense for count_person?

Reply:
done!